### DISCUSSION TITLE
algebra relazionale - 9

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#66] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/6a1ed5a5-ef1f-478c-985d-8f6ca892ab90)

**Comment by daromacarlo** [comment]
domanda) Il professore vuole tutto in una riga o va bene così?

![Immagine 2024-01-16 163131](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/f8d70f5f-f2bf-429f-8306-6ccaa2de4ee2)

**Reply by IonutCicio** [reply]
Il prof. preferisce decisamente se lo fai a più step. Anzi, ti suggerirei di dare nomi più significativi alle varibili, così ci lavori più facilmente te, e per il prof. è meno faticoso da correggere.

**Reply by daromacarlo** [reply]
ok, grazie mille.

**Reply by IonutCicio** [reply]
Attento alla prima query, chiede che la stanza deve essere disponibile per l'**intero** periodo, non una parte del periodo. La soluzione ufficiale è questa 

![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/55fdd025-3b1c-4a1a-8891-30f0b4737438)

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/7667c789-d869-4e67-b09d-8f8f25be7823)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/ea43f3b1-60ae-4a76-8af1-dbaf1e05fb4e)
